plutus-use-cases/test/Spec/Lib.hs

```diff
@@ -25,3 +33,14 @@ reasonable (Ledger.ValidatorScript s) maxSize = do
 
 goldenPir :: FilePath -> CompiledCode a -> TestTree
 goldenPir path code = goldenVsString "PIR" path (pure $ fromString $ show $ pretty $ fromJust $ getPir code)
+
+staticFee :: Integer
+staticFee = 0
+
+timesFeeAdjust :: Integer -> Integer -> Value
```

Comment:
Comments please. I'm not entirely sure what these are for tbh.

Reply:
Added some comments. Here is the story:

When I was verifying the totals I got code like, the wallet should have 14 Ada. But it is not very clear (in the presence of fees) how you come up with that 14 (was there one tx fee, two, more?). I use this function mostly to make clear what value is spent due to fees.

I've added some comments.

Reply:
Ah I see. Yes, it's a bit annoying to write assertions modulo fees. I wonder if we could just make the fees small enough that we could assert fee balances up to some epsilon small enough to cover feeds but not enough to affect our tests?